src/crystal/scheduler.cr

```diff
@@ -57,9 +57,15 @@ class Crystal::Scheduler
   end
 
   protected def resume(fiber : Fiber) : Nil
+    GC.disable
```

Comment:
I wonder, would it be a good idea to have a block version of `GC.disable` that reenables GC when the block is exited?

Reply:
Due to how things work, I would prefer to not use a block version. This `GC.disable` is paired with the enable of the `Fiber#run` or the trailing `GC.enable` in `Scheduler#resume` depending on whether the `fiber` argument is new or not.